src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -178,11 +179,79 @@ public void applyMergeReview(MergeReview mergeReview) throws ServiceException {
 		final Set<Long> conflictingConceptIds = getConflictingConceptIds(mergeReview);
 		final Set<Long> conflictsRemaining = new HashSet<>(conflictingConceptIds);
 		conflictsRemaining.removeAll(manuallyMergedConceptIds);
+		conflictsRemaining.removeIf(conflict -> {
+			// Silently restore components if innocently altered, i.e. edited after content was versioned.
+			String id = conflict.toString();
+			String mergeReviewId = mergeReview.getId();
+			Concept parent = conceptService.find(id, mergeReview.getSourcePath());
+			Concept child = conceptService.find(id, mergeReview.getTargetPath());
+
+			// Versioned on parent whilst simultaneously deleted on child
+			if (parent != null && parent.isReleased() && child == null) {
+				// Simply restore parent
+				manuallyMergedConcepts.add(new ManuallyMergedConcept(mergeReviewId, conflict, asJson(parent), false));
+
+				return true;
+			}
+
+			// Child has diverged from parent
+			if (parent != null && parent.isReleased() && parent.getEffectiveTime() != null && child != null && !child.isReleased() && child.getEffectiveTime() == null) {
+				// Find difference in Descriptions and restore
+				Set<Description> parentDescriptions = parent.getDescriptions();
+				Set<Description> childDescriptions = child.getDescriptions();
+
+				// Descriptions removed in child
+				Set<Description> parentDescriptionsDeletedInChild = new HashSet<>(parentDescriptions);
+				parentDescriptionsDeletedInChild.removeAll(childDescriptions);
+
+				// Descriptions added in child
+				Set<Description> childDescriptionsAddedInChild = new HashSet<>(childDescriptions);
+				childDescriptionsAddedInChild.removeAll(parentDescriptions);
+
+				// Restore parent Descriptions
+				for (Description description : parentDescriptionsDeletedInChild) {
+					parent.addDescription(description);
+				}
+
+				// Keep child Descriptions
+				for (Description description : childDescriptionsAddedInChild) {
+					parent.addDescription(description);
+				}
+
+				// Find difference in Axioms and restore
+				Set<Axiom> parentAxioms = parent.getClassAxioms();
+				Set<Axiom> childAxioms = child.getClassAxioms();
+
+				// Axioms removed in child
+				Set<Axiom> parentAxiomsDeletedInChild = new HashSet<>(parentAxioms);
+				parentAxiomsDeletedInChild.removeAll(childAxioms);
+
+				// Axioms added in child
+				Set<Axiom> childAxiomsAddedInChild = new HashSet<>(childAxioms);
+				childAxiomsAddedInChild.removeAll(parentAxioms);
+
+				// Restore parent Axioms
+				for (Axiom axiom : parentAxiomsDeletedInChild) {
+					parent.addAxiom(axiom);
+				}
+
+				// Keep child Axioms
+				for (Axiom axiom : childAxiomsAddedInChild) {
+					parent.addAxiom(axiom);
+				}
+
+				manuallyMergedConcepts.add(new ManuallyMergedConcept(mergeReviewId, conflict, asJson(parent), false));
+
+				return true;
+			}
+
+			return false;
+		});
```

Comment:
@dmcgihtsdo From the logic implemented here(Child has diverged from parent), it could be simpler to just include the "added" components on task. (i.e components exist on the task only)

Reply:
Thanks @CoderMChu 

Do you mean instead of setting the task's data on the parent, set the parent's data on the child? For example, instead of this:

`
... 
parent.addAxiom(axiom);
...
manuallyMergedConcepts.add(new ManuallyMergedConcept(mergeReviewId, conflict, asJson(parent), false));
`

Do this: 
`
... 
child.addAxiom(axiom);
...
manuallyMergedConcepts.add(new ManuallyMergedConcept(mergeReviewId, conflict, asJson(child), false));
`

Reply:
@dmcgihtsdo No. I mean those steps to store components deleted on the task back into parent version. e.g (// Descriptions removed in child)